package.json

```
@@ -0,0 +1,90 @@
+{
+  "name": "@wfcd/profile-parser",
+  "version": "0.0-dev",
+  "description": "Parser for Digital Extreme's profile data",
+  "type": "module",
+  "main": "src/ProfileParser.js",
```

something like this so you can use the module reference in tests instead of referencing the files themselves... makes it more test-like

```suggestion
  "main": "src/ProfileParser.js",
  "exports": {
    ".": {
      "import": "src/ProfileParser.js"
      "type": "types/src/ProfileParser.d.ts"
    }
  }, 
```